extension-manifest-v2/src/utils/matcher.js

```diff
@@ -81,12 +81,19 @@ export function isBug(details) {
 	const { engine } = bugDb;
 
 	const request = fromWebRequestDetails(details);
+	let isFilterBlocker = false;
 
-	const matches = engine.getPatternMetadata(request);
+	let matches = engine.getPatternMetadata(request);
+
+	if (matches.length > 0) {
+		isFilterBlocker = true;
```

Comment:
Maybe there is a better name than "isFilterBlocker". When it is used, it is referred to as "isFilterMatched".

I wonder if it should be rather should like "isInPatternDB"/"existsInPatternDb"/"hasPatternRule"/"foundInPatternMetadata". I'm lacking an intuition what it means to have a non-empty match in the pattern meta.

Or maybe "blockingShouldBeHandledByPatternEngine" / "usePatternEngineForBlocking"?

Reply:
all good points. I'm also not sure how to name it best, but my hope for this code is to be short lived. I will go with `isFilterMatched` for now